test/FilterPluginManagerCompatibilityTest.php

```diff
@@ -5,53 +5,84 @@
 namespace LaminasTest\Filter;
 
 use Generator;
-use Laminas\Filter\Exception\RuntimeException;
-use Laminas\Filter\FilterInterface;
 use Laminas\Filter\FilterPluginManager;
-use Laminas\ServiceManager\ServiceManager;
-use Laminas\ServiceManager\Test\CommonPluginManagerTrait;
+use Laminas\ServiceManager\Exception\InvalidServiceException;
 use PHPUnit\Framework\TestCase;
-use ReflectionProperty;
+use ReflectionClass;
+use stdClass;
+use Throwable;
 
+use function assert;
+use function class_exists;
+use function is_string;
 use function strpos;
 
 class FilterPluginManagerCompatibilityTest extends TestCase
 {
-    use CommonPluginManagerTrait;
```

Comment:
did we drop this on purpose? is that due to the `callable` type?

Reply:
Yes, and also that trait only works with `SingleInstancePluginManager`